src/Microsoft.CSharp/tests/Microsoft.CSharp.Tests.builds

```diff
@@ -2,7 +2,11 @@
 <Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
-    <Project Include="Microsoft.CSharp.Tests.csproj" />
+    <Project Include="Microsoft.CSharp.Tests.csproj"/>
```

Comment:
I assume this is because we don't want to run netcore50 and net46 on non-windows but if that is the case we should potentially look into other options.